src/interfaces.d.ts

```diff
@@ -0,0 +1,60 @@
+export interface User {
```

Comment:
In a `.d.ts` file you don't need to export, you can just `declare` the interface 
```suggestion
declare interface User {
```